README.md

```diff
@@ -28,6 +28,14 @@ The following packages are required to use the caver-js library.
 - [npm](https://www.npmjs.com/get-npm)
 - [gcc-c++](https://gcc.gnu.org/)
 
+**Note** The recommended node versions are
+- lts/carbon (8.16.0)
+- lts/dubnium (10.16.0)
+
+also, support [AWS lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).
```

Comment:
```suggestion
also, support [AWS lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).
```

여기도 지원한다는 것이 무슨 의미인지 불명확하네요. caver-js를 AWS lambda runtimes에서 쓸수 있다는건데, 어떻게 쓸수 있는지, 대략적인 설명이 있으면 좋을 것 같아요. 람다는 버전 몇을 지원하는데 어떻게 쓰면 된다는 간단한 설명이 있으면 좋을 것 같네요.

Reply:
Since the Node.js versions supported by Lambda are 8 and 10 (https://docs.aws.amazon.com/en_us/lambda/latest/dg/programming-model.html), the statement that supports lambda May be omitted from this document.
After removing lambda, based on the test results, i described caver-js can be run on node 8 or 10, and recommended version is based on lts.